### PR TITLE
fix error when were hidden exception from docker.run

### PR DESCRIPTION
**Pull request by makar21**

### CODE REVIEW

File: tatau_core/sessions/docker.py [hunk 1]
```diff
@@ -113,5 +113,5 @@ def _run(self, *args, image=None, bind_data=True, **kwargs):
         try:
             return self.docker_client.containers.run(image, *args, **options)
         except ContainerError as err:
-            logger.exception('Container output: {}'.format(err.container.logs().decode('utf-8')))
+            logger.exception('Container error: {}'.format(err))
```
Comment (Yulya) [inline tatau_core/sessions/docker.py]: I don't understand this change.
We don't we need logs here?

Comment (Yulya) [reply tatau_core/sessions/docker.py]: So assuming we can have empty logs from container and then nothing goes to log lets leave logging of both container.logs and error here.
Container logs can be useful for us to identify issue that happened

Comment (makar21) [reply tatau_core/sessions/docker.py]: Call of err.container.logs() rais exception and we dont catch real exception - err.

Comment (Yulya) [reply tatau_core/sessions/docker.py]: Oh, ok.
So maybe log err and after try to take logs? 
Or find a way to get logs in a "safe" way?

**Pull request merged**